rotkehlchen/chain/ethereum/utils.py

```diff
@@ -41,29 +41,59 @@
 MULTICALL_CHUNKS = 20
 
 
-def token_normalized_value_decimals(token_amount: int, token_decimals: int | None) -> FVal:
-    if token_decimals is None:  # if somehow no info on decimals ends up here assume 18
-        token_decimals = 18
+@overload
+def _normalize_value(
+        amount: int | FVal,
+        decimals: int | None,
+        as_int: Literal[True],
+) -> int:
+    ...
 
-    return token_amount / (FVal(10) ** FVal(token_decimals))
 
+@overload
+def _normalize_value(
```

Comment:
Would it not make more sense to have two different functions to avoid the extra if check? Since this is a function deep in our call stack that's used a lot. Some something like normalize_fval and normalize_int